lib/auth/auth_with_roles.go

```diff
@@ -808,6 +809,15 @@ func (a *AuthWithRoles) NewKeepAliver(ctx context.Context) (services.KeepAliver,
 }
 
 func (a *AuthWithRoles) GenerateUserCert(key []byte, username string, ttl time.Duration, compatibility string) ([]byte, error) {
+	certs, err := a.GenerateUserCertSet(key, username, ttl, compatibility)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	return certs.Ssh, nil
```

Comment:
`SSH` here and below, according to abbreviations in https://github.com/golang/go/wiki/CodeReviewComments